wire-ios-data-model/Source/MLS/CommitSender.swift

```diff
@@ -151,7 +151,7 @@ public actor CommitSender: CommitSending {
         do {
             WireLogger.mls.info("merging commit for group (\(groupID.safeForLoggingDescription))")
             // TODO: [WPB-5829] handle buffered messages
-            let bufferedDecryptedMessages = try await coreCrypto.perform {
+            _ = try await coreCrypto.perform {
```

Comment:
can we remove the TODO? cc @typfel the ticket is done.

Reply:
According to @typfel these TODOs can simply be deleted.
We will not run into a scenario where we need to handle buffered decrypted messages, because sending a commit and decrypting a message are non-rentrant operations and therefore we will never attempt to decrypt a message while sending a commit.
Please confirm @typfel.

Reply:
@caldrian we can remove the todo then and add this comment at the same place